app/views/catalog/_zero_results.html.erb

```diff
@@ -1,5 +1,10 @@
 <h1><%= t('trln_argon.search.zero_results.title', local_institution_name: institution_long_name) %></h1>
 
+<% if filters_applied? %>
+   <% remove_applied_facets %>
+   <h3><%= link_to 'Try removing your search filters', search_state.params %> to see more results.</h3>
```

Comment:
The display text should be in a translation file to make overriding the language of the link easy: `config/locales/trln_argon.en.yml`

I think I would do something like this directly in the template:

`<%= link_to t('trln_argon.search.zero_results.remove_filters'), url_for(search_state.params_for_search(f: nil)) %>`